<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>1200</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
	background-color: rgb(240, 240, 240);
}

QPushButton {
	background-color: rgb(230, 230, 230);
	color: rgb(0, 0, 0);
	border: none;
	border-bottom: 1px solid rgb(155, 155, 155);
	border-radius: 4px;
}

QPushButton:hover {
	background-color: rgb(235, 235, 235);
	border-bottom: 1px solid rgb(175, 175, 175);
}

QPushButton:pressed {
	background-color: rgb(220, 220, 220);
	border-bottom: 1px solid rgb(220, 220, 220);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="2,1,1,1,1,1">
    <property name="spacing">
     <number>4</number>
    </property>
    <property name="leftMargin">
     <number>4</number>
    </property>
    <property name="topMargin">
     <number>4</number>
    </property>
    <property name="rightMargin">
     <number>4</number>
    </property>
    <property name="bottomMargin">
     <number>4</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>192</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>16</number>
       </property>
       <property name="topMargin">
        <number>16</number>
       </property>
       <property name="rightMargin">
        <number>16</number>
       </property>
       <property name="bottomMargin">
        <number>16</number>
       </property>
       <item>
        <widget class="QLabel" name="label_operation">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>1</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>14</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_result">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>1</horstretch>
           <verstretch>2</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>32</pointsize>
           <italic>false</italic>
           <bold>false</bold>
           <underline>false</underline>
           <strikeout>false</strikeout>
          </font>
         </property>
         <property name="text">
          <string notr="true">0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>4</number>
      </property>
      <item>
       <widget class="QPushButton" name="pushButton_clear">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>1</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>18</pointsize>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
          <stylestrategy>PreferDefault</stylestrategy>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-color: rgb(200, 200, 200);
}

QPushButton:hover {
	background-color: rgb(205, 205, 205);
	border-bottom: 1px solid rgb(175, 175, 175);
}

QPushButton:pressed {
	background-color: rgb(190, 190, 190);
	border-bottom: 1px solid rgb(190, 190, 190);
}</string>
        </property>
        <property name="text">
         <string>AC</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_sign">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>1</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>18</pointsize>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
          <stylestrategy>PreferDefault</stylestrategy>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-color: rgb(200, 200, 200);
}

QPushButton:hover {
	background-color: rgb(205, 205, 205);
	border-bottom: 1px solid rgb(175, 175, 175);
}

QPushButton:pressed {
	background-color: rgb(190, 190, 190);
	border-bottom: 1px solid rgb(190, 190, 190);
}</string>
        </property>
        <property name="text">
         <string>+/-</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_percent">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>1</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>18</pointsize>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
          <stylestrategy>PreferDefault</stylestrategy>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-color: rgb(200, 200, 200);
}

QPushButton:hover {
	background-color: rgb(205, 205, 205);
	border-bottom: 1px solid rgb(175, 175, 175);
}

QPushButton:pressed {
	background-color: rgb(190, 190, 190);
	border-bottom: 1px solid rgb(190, 190, 190);
}</string>
        </property>
        <property name="text">
         <string>%</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_division">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>1</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>18</pointsize>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
          <stylestrategy>PreferDefault</stylestrategy>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-color: rgb(255, 140, 0);
}

QPushButton:hover {
	background-color: rgb(255, 150, 0);
	border-bottom: 1px solid rgb(175, 175, 175);
}

QPushButton:pressed {
	background-color: rgb(245, 130, 0);
	border-bottom: 1px solid rgb(245, 130, 0);
}</string>
        </property>
        <property name="text">
         <string>/</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>4</number>
      </property>
      <item>
       <widget class="QPushButton" name="pushButton_7">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>1</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>18</pointsize>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
          <stylestrategy>PreferDefault</stylestrategy>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="text">
         <string>7</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_8">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>1</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>18</pointsize>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
          <stylestrategy>PreferDefault</stylestrategy>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="text">
         <string>8</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_9">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>1</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>18</pointsize>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
          <stylestrategy>PreferDefault</stylestrategy>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="text">
         <string>9</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_multiply">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>1</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>18</pointsize>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
          <stylestrategy>PreferDefault</stylestrategy>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-color: rgb(255, 140, 0);
}

QPushButton:hover {
	background-color: rgb(255, 150, 0);
	border-bottom: 1px solid rgb(175, 175, 175);
}

QPushButton:pressed {
	background-color: rgb(245, 130, 0);
	border-bottom: 1px solid rgb(245, 130, 0);
}</string>
        </property>
        <property name="text">
         <string>x</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>4</number>
      </property>
      <item>
       <widget class="QPushButton" name="pushButton_4">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>1</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>18</pointsize>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
          <stylestrategy>PreferDefault</stylestrategy>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="text">
         <string>4</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_5">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>1</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>18</pointsize>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
          <stylestrategy>PreferDefault</stylestrategy>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="text">
         <string>5</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_6">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>1</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>18</pointsize>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
          <stylestrategy>PreferDefault</stylestrategy>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="text">
         <string>6</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_minus">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>1</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>18</pointsize>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
          <stylestrategy>PreferDefault</stylestrategy>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-color: rgb(255, 140, 0);
}

QPushButton:hover {
	background-color: rgb(255, 150, 0);
	border-bottom: 1px solid rgb(175, 175, 175);
}

QPushButton:pressed {
	background-color: rgb(245, 130, 0);
	border-bottom: 1px solid rgb(245, 130, 0);
}</string>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="spacing">
       <number>4</number>
      </property>
      <item>
       <widget class="QPushButton" name="pushButton_1">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>1</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>18</pointsize>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
          <stylestrategy>PreferDefault</stylestrategy>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>1</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>18</pointsize>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
          <stylestrategy>PreferDefault</stylestrategy>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="text">
         <string>2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>1</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>18</pointsize>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
          <stylestrategy>PreferDefault</stylestrategy>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="text">
         <string>3</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_plus">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>1</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>18</pointsize>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
          <stylestrategy>PreferDefault</stylestrategy>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-color: rgb(255, 140, 0);
}

QPushButton:hover {
	background-color: rgb(255, 150, 0);
	border-bottom: 1px solid rgb(175, 175, 175);
}

QPushButton:pressed {
	background-color: rgb(245, 130, 0);
	border-bottom: 1px solid rgb(245, 130, 0);
}</string>
        </property>
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,1">
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="pushButton_0">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>1</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <property name="spacing">
         <number>4</number>
        </property>
        <item>
         <widget class="QPushButton" name="pushButton_dot">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>1</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>18</pointsize>
           </font>
          </property>
          <property name="text">
           <string>,</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_result">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>1</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>18</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
	background-color: rgb(255, 140, 0);
}

QPushButton:hover {
	background-color: rgb(255, 150, 0);
	border-bottom: 1px solid rgb(175, 175, 175);
}

QPushButton:pressed {
	background-color: rgb(245, 130, 0);
	border-bottom: 1px solid rgb(245, 130, 0);
}</string>
          </property>
          <property name="text">
           <string>=</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
